tests/index.js

```diff
@@ -1053,6 +1053,25 @@ describe('apply function', () => {
         });
     });
 
+    describe('with simple string patterns', () => {
+        it('can move multiple files', (done) => {
```

Comment:
Wording could be better if anyone has suggestions.

Reply:
```js
describe('Patterns - {String}', () => { 
   it('Multiple Files', () => {})
})
```

But 🐦 , all good